pkg/controller.v1/common/job.go

```diff
@@ -357,3 +361,8 @@ func (jc *JobController) CleanupJob(runPolicy *apiv1.RunPolicy, jobStatus apiv1.
 func (jc *JobController) calcPGMinResources(minMember int32, replicas map[apiv1.ReplicaType]*apiv1.ReplicaSpec) *v1.ResourceList {
 	return CalcPGMinResources(minMember, replicas, jc.PriorityClassLister.Get)
 }
+
+func (jc *JobController) JobSuspended(job interface{}) (*bool, error) {
+	log.Infof("Not implemented.")
```

Comment:
I am wondering if we should merge this since the feature is not completed.

Reply:
This is a base class default function in case Job subclasses(TFJob, MPIJob, etc.) do not implement this method.

Reply:
Actually it will be override in Job subclass which supports job suspend.

Reply:
If `DeletePodsAndServices` has an implementation, I don't see why this function wouldn't have one.